### ISSUE TITLE
Large set of human ribosomal proteins annotated to "cadherin binding involved in cell-cell adhesion"

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1510]
From
http://amigo.geneontology.org/amigo/reference/PMID:25468996

This is also generating process annotations to
	RPL29	60S ribosomal protein L29		
"cell-cell adhesion"		
GOC	Homo sapiens	IEA	GO:0098641	60s ribosomal protein l29 GO_REF:0000108	20161126

via the  GO F-P pipeline

and by other pipelines being propagated across many metazoan species.

I don't know how many of these really bind cadherin, but a lot are likely false positives out of this 296.

**Comment by ValWood** [comment]
@RLovering

**Comment by RLovering** [comment]
@rachhuntley

**Comment by rachhuntley** [comment]
Originated from this GO help ticket http://jira.geneontology.org/browse/GO-1445?filter=-4

Copied my comment below to continue the thread here:
These are my annotations. I made them in consultation with the author based on two experiments. I will contact her to see if she has any thoughts on those we should exclude or see if she's done any follow-up experiments. In the meantime, Val could you send me the list of outliers that you have found?

**Comment by ValWood** [comment]
RPL36AL
RPL6
RPL15
(also annotated to nucleus from this paper, which also seems odd?)

Then
RPL24
EEF1G Eukaryotic translation initiation factor 1G
ENO1 enolase
NOP56
EIF4H Eukaryotic translation initiation factor 4H
CCNB2 mitotic cyclin
RPL6
EIF2A Eukaryotic translation initiation factor 2A
RSL1D1	Ribosomal L1 domain-containing protein 1
RPL24	60S ribosomal protein L24
EEF1G	Elongation factor 1-gamma
HSPA8	Heat shock cognate 71 kDa protein	
PSMB6	Proteasome subunit beta type-6
HSP90AB1	Heat shock protein HSP 90-beta	
DHX29	ATP-dependent RNA helicase DHX29
EIF5	Eukaryotic translation initiation factor 5	
GCN1	eIF-2-alpha kinase activator GCN1
ATIC	Bifunctional purine biosynthesis protein PURH
ALDOA	Fructose-bisphosphate aldolase A
EEF2	Elongation factor 2
RPL14	60S ribosomal protein L14
EIF3E	Eukaryotic translation initiation factor 3 subunit E
H1FX	Histone H1x
EEF1D	Elongation factor 1-delta
MACF1	Microtubule-actin cross-linking factor 1, isoforms 1/2/3/5	
PFN1	Profilin-1
IDH1	Isocitrate dehydrogenase [NADP] cytoplasmic	
RPS2	40S ribosomal protein S2
MYO1B	Unconventional myosin-Ib	
RPL29	60S ribosomal protein L29
RPL34	60S ribosomal protein L34
RPS26	40S ribosomal protein S26
RPL15	60S ribosomal protein L15	
MYO6	Unconventional myosin-VI
RARS	Arginine--tRNA ligase, cytoplasmic	
RPL7A	60S ribosomal protein L7a
HSPA1A	Heat shock 70 kDa protein 1A
EIF2S3	Eukaryotic translation initiation factor 2 subunit 3	
RPL23A	60S ribosomal protein L23a
EIF4G1	Eukaryotic translation initiation factor 4 gamma 1	
MYH9	Myosin-9

Probably more. These are just the ones that I know are highly expressed and common contaminants in Mass Spec  experiments from fission yeast.
What is the assay for "cell-cell adhesion"? is it only inferred from the binding.Seems like a stretch?

Val

MRE11A would be strange too
CCNB2 mitotic cyclin

**Comment by rachhuntley** [comment]
Hi Val,

I have looked at this paper again and realised that the protein list the author pointed me to was not quite right, there was an additional localisation experiment that should have been taken into account. I have now updated the annotations and there are far fewer proteins annotated to cadherin binding involved in cell-cell adhesion and cell-cell adherens junction.
I haven't checked all on your list, but a lot of them are now changed.

Thanks for alerting us to this.

Rachael.
